test/integration/targets/setup_paramiko/install-MacOSX-10-python-3.yml

```diff
@@ -0,0 +1,6 @@
+- name: Install Paramiko for Python 3 on MacOS
+  pip: # no homebrew package manager in core, just use pip
+    name: paramiko
```

Comment:
Same thing here. Do we want to pin this to a specific version?

Reply:
It's not pinned in any of the other files for this target either. I suppose we could, but I'm going to avoid that in this PR.